chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/MetricEventPublisher.java

```diff
@@ -25,7 +25,8 @@ public void publishMetricEvent(MetricType metricType, String... tags) {
     }
 
     public void publishMetricEvent(MetricType metricType, AtomicInteger atomicTimeoutGauge) {
-        MetricEvent metricEvent = new MetricEvent(this, metricType, atomicTimeoutGauge);
+        //MetricEvent constructor has been deprecated. So falling back to the internally used Constructor
```

Comment:
I noticed that all constructors are public in `MetricEvent` and this not "internally". I think we can remove that comment if that's what you wanted to point out here. If not, please elaborate a bit on what the comment explains.